cfme/tests/webui/test_general_ui.py

```diff
@@ -465,6 +467,39 @@ def test_add_provider_button_accordion(has_no_providers, provider):
     assert view.add_button.is_displayed
 
 
+@pytest.mark.tier(1)
+@pytest.mark.meta(automates=[1642948])
+@pytest.mark.provider([VMwareProvider], selector=ONE)
```

Comment:
**Suggestion:**
As going through BZ, if possible can we check with one of Cloud Provider.? like AWS or OSP etc.